Dockerfile

```diff
@@ -39,7 +39,7 @@ COPY app/package.json /headlamp/app/package.json
 COPY frontend/package*.json /headlamp/frontend/
 COPY frontend/patches/* /headlamp/frontend/patches/
 WORKDIR /headlamp
-RUN cd ./frontend && npm install --only=prod
+RUN cd ./frontend && npm install
```

Comment:
vite is a dev dependency and we need it to build the frontend/
so --only=prod was removed